addons/l10n_in/models/account_invoice.py

```diff
@@ -30,6 +30,8 @@ class AccountMove(models.Model):
     l10n_in_shipping_port_code_id = fields.Many2one('l10n_in.port.code', 'Port code')
     l10n_in_reseller_partner_id = fields.Many2one('res.partner', 'Reseller', domain=[('vat', '!=', False)], help="Only Registered Reseller")
     l10n_in_journal_type = fields.Selection(string="Journal Type", related='journal_id.type')
+    l10n_in_reseller_partner_industry_id = fields.Many2one(related="l10n_in_reseller_partner_id.industry_id")
+    l10n_in_reseller_partner_gstin = fields.Char("Reseller GSTIN ", related="l10n_in_reseller_partner_id.vat")
```

Comment:
It does not make much sense if these fields are not shown in the view.

Reply:
But we need them for reports to be used in domains